src/API.jl

```diff
@@ -170,7 +170,7 @@ function up(ctx::Context, pkgs::Vector{PackageSpec};
 end
 
 resolve(ctx::Context=Context()) =
-    up(ctx, level=UPLEVEL_FIXED, mode=PKGMODE_MANIFEST, update_registry=false)
```

Comment:
Why change this?

Reply:
Passing `PKGMODE_MANIFEST` to `up` will cause `Pkg` to fix all dependencies at their current version, not just the direct dependencies.

Reply:
But isn't that what resolve should do?

Reply:
Hmm, perhaps I misunderstand the intention behind `resolve`. In any case, that is really a separate issue (we should maybe have a test which checks the difference). I will change it change it back for the sake of merging this PR sooner.